app/src/main/java/com/arctouch/codechallenge/api/TmdbApi.kt

```diff
@@ -27,8 +27,7 @@ interface TmdbApi {
     fun upcomingMovies(
```

Comment:
you can easily remove this line too: 

```const val DEFAULT_REGION = "BR"```

Reply:
How would be a better approach to do not duplicate the API_KEY usage?

Reply:
We could use an http Interceptor to add a query paramenter with API key.
Implemented in [#ca8b5733](https://github.com/diogojs/tmdb_app/pull/1/commits/ca8b5733)